components/booking/AvailableTimes.tsx

```diff
@@ -84,7 +91,10 @@ const AvailableTimes: FC<AvailableTimesProps> = ({
               <div key={slot.time.format()}>
                 <Link href={bookingUrl}>
                   <a
-                    className="block py-4 mb-2 font-medium bg-white border rounded-sm dark:bg-gray-600 text-primary-500 dark:text-neutral-200 border-brand dark:border-transparent hover:text-white hover:bg-brand hover:text-brandcontrast dark:hover:border-black dark:hover:bg-black"
+                    className={classNames(
+                      "block py-4 mb-2 font-medium bg-white border rounded-sm dark:bg-gray-600 text-primary-500 dark:text-neutral-200 dark:border-transparent hover:text-white hover:bg-brand hover:text-brandcontrast dark:hover:border-black dark:hover:bg-brand dark:hover:text-brandcontrast",
+                      brand === "#fff" || brand === "#fffff" ? "border-brandcontrast" : "border-brand"
```

Comment:
```suggestion
                      brand === "#fff" || brand === "#ffffff" ? "border-brandcontrast" : "border-brand"
```
Did you miss an extra `f` in the colour code?

Reply:
Good catch, I did. Thanks